src/screens/ContactScreen/index.js

```diff
@@ -317,10 +385,9 @@ const ContactScreen = ({navigation}) => {
     if (changedText.length > 0) {
       debounced(changedText);
     } else {
+      setUsersSearch([]);
       debounced.cancel();
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onSearchTextChange` function has been updated to clear `usersSearch` when the search text is empty. This is a good change as it ensures that old search results are not shown when there is no search query.



<!-- This is an auto-generated comment by OSS CodeRabbit -->